File: executable/r/basic-box-plot.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

y0 = rnorm(50)
y1 = rnorm(50)+1

trace1 <- list(
  y = y0, 
  type = "box"
)
trace2 <- list(
  y = y1, 
  type = "box"
)
data <- list(trace1, trace2)

response <- p$plotly(data, kwargs=list(filename="basic-box-plot", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/basic-box-plot.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

y0 = Math.random() * 50
y1 = Math.random() * 50 + 1

var trace1 = {
  y: y0, 
  type: "box"
};
var trace2 = {
  y: y1, 
  type: "box"
};
var data = [trace1, trace2];

var graph_options = {filename: "basic-box-plot", fileopt: "overwrite"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});